src/tools/ipv6-ula-generator/index.ts

```diff
@@ -0,0 +1,16 @@
+import { BuildingFactory } from '@vicons/tabler';
+import { defineTool } from '../tool';
+
+export const tool = defineTool({
+  name: 'IPv6 ULA generator',
+  path: '/ipv6-ula-generator',
+  description: `When you set up IPv6, you may need to set up your own local, non-routable IP addresses on your network - 
+similar to the older RFC1918 blocks. RFC4193 addresses (no pun intended) this issue.
+
+Your IP address block assigned should be generated randomly from the fc00::/7 block. As IETF hasn't formalized fc00::/8, 
+addresses should be assigned out of fd00::/8.`,
```

Comment:
Please don't reuse other [website](https://cd34.com/rfc4193/) descriptions, it's not really faire, and will impact SEO

Thus, this description is used for the description meta, which is what is displayed on search engine results, so its preferable to have something appealing.

If you find relevent to add more complete information, please add them directly in the vue file of the tool

```suggestion
  description: `Generate your personnal ipv6 ULA (Unique Local Address) from your mac address.`,
```